packages/transaction-controller/src/helpers/IncomingTransactionHelper.test.ts

```diff
@@ -187,6 +187,7 @@ describe('IncomingTransactionHelper', () => {
         const helper = new IncomingTransactionHelper({
           ...CONTROLLER_ARGS_MOCK,
           remoteTransactionSource,
+          getCurrentAccount: () => undefined as unknown as string,
```

Comment:
Not sure why we are setting this as `undefined` here?

The goal of this test is to check we query everything if `getLastFetchedBlockNumbers` returns nothing which is the default.

If we want to test an empty `getCurrentAccount` specifically, shall we create an alternate unit test?

Reply:
You are right, I pushed a new test case for this scenario. https://github.com/MetaMask/core/pull/3741/commits/63b8a04748eb30dd48c9978aaadcd6cd9790acfb